packages/bundler/src/Bundler.ts

```diff
@@ -42,19 +66,61 @@ export class Bundler implements IBundler {
      * @returns Promise<UserOpResponse>
      */
     async sendUserOp(userOp: UserOperation, chainId: ChainId): Promise<UserOpResponse> {
-        const hexifiedUserOp = deepHexlify(await resolveProperties(userOp))        
-        let params = [hexifiedUserOp, this.bundlerConfig.epAddress, chainId]
-        const response: UserOpResponse = await sendRequest({
-            url: `${this.bundlerConfig.bundlerUrl}`,
+        // transformUserOP will convert all bigNumber values to string
+        userOp = transformUserOP(userOp)
+        const hexifiedUserOp = deepHexlify(await resolveProperties(userOp))
+        let params = [hexifiedUserOp, this.bundlerConfig.epAddress]
+        const sendUserOperationResponse: SendUserOperationResponse = await sendRequest({
+            url: `${this.bundlerConfig.bundlerUrl}/${chainId}/${this.bundlerConfig.apiKey}`,
             method: HttpMethod.Post,
             body: {
                 method: 'eth_sendUserOperation',
                 params: params,
                 id: getTimestampInSeconds(),
                 jsonrpc: '2.0'
             }
-        })
-        console.log('bundler response', response)
+        })        
+        let response: UserOpResponse = {
+            userOpHash: sendUserOperationResponse.result,
+            wait: (): Promise<UserOpReceipt> => {
```

Comment:
wait method has optional confirmations parameter as well. We need to use that as well while resolving the promise.